### PR TITLE
wiki103_quality 7 Fixed remaining rows - corrected invalid answers

### PR DESCRIPTION
**Pull request by Gosheni**

### CODE REVIEW

File: src/utils/clean_csv.py
Comment (klxu03) [inline src/utils/clean_csv.py:1]: can you undo your changes in the clean_csv.py file?

**Review by klxu03 — CHANGES_REQUESTED** [review]
Fix clean_csv

File: src/utils/clean_csv.py
Comment (Gosheni) [reply src/utils/clean_csv.py:1]: Ok

**Review by klxu03 — APPROVED** [review]
lgtm

**Pull request merged**